<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCJLUClientMainWindow</class>
 <widget class="QMainWindow" name="CCJLUClientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCJLUClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_show_state">
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_login">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>274</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="action_create_user"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>登录</string>
    </property>
    <addaction name="action_login"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>日志</string>
    </property>
    <addaction name="action_check_logger"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_helpWindow"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>最小化</string>
    </property>
    <addaction name="action_mini"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>退出</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_login">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="action_check_logger">
   <property name="text">
    <string>查看日志</string>
   </property>
  </action>
  <action name="action_create_user">
   <property name="text">
    <string>创建用户</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_helpWindow">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="action_mini">
   <property name="text">
    <string>隐藏到托盘</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
